ra2ce/graph/hazard/hazard_files.py

```diff
@@ -0,0 +1,11 @@
+from pathlib import Path
+from typing import Optional
+
+from attr import dataclass
+
+
+@dataclass
+class HazardFiles:
+    tif: Optional[list[Path]] = None
+    shp: Optional[list[Path]] = None
+    table: Optional[list[Path]] = None
```

Comment:
It does not make much sense to me to have optional lists, as  discussed in other PR, just make it default as empty rather than Optional